app/build.gradle

```diff
@@ -62,15 +62,21 @@ android {
 }
 
 dependencies {
-	testImplementation 'junit:junit:4.13.2'
-
+	// Kotlin
 	implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 	implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4"
+
+	// Support
 	implementation "com.android.support:appcompat-v7:$support_version"
 	implementation "com.android.support:design:$support_version"
 	implementation "com.android.support:preference-v7:$support_version"
 	implementation "com.android.support:preference-v14:$support_version"
+
+	// External
 	implementation 'com.github.markusfisch:CameraView:1.9.2'
 	implementation 'com.github.markusfisch:ScalingImageView:1.4.1'
-	implementation 'com.github.markusfisch:zxing-cpp:v2.2.0.1'
+	implementation 'com.github.markusfisch:zxing-cpp:v2.1.0.2'
```

Comment:
That's the wrong version 😬 If you use `v2.2.0.1`, it should compile and run on your system, too. This is what is specified on `master`.

Reply:
I made a small mistake while rebasing because I manually checked what was conflicting and updated them 😁